honeybee_schema/radiance/state.py

```diff
@@ -0,0 +1,78 @@
+"""State Schema"""
+
+from pydantic import Field, constr
+from typing import List
+from .._base import NoExtraBaseModel, IDdBaseModel
+
+from ..geometry import Face3D
+
+
+class StateGeometryAbridged(IDdBaseModel):
+    """StateGeometryAbridged represents shading for States."""
+
+    type: constr(regex='^StateGeometryAbridged$') = 'StateGeometryAbridged'
+
+    modifier: str = Field(
+        default=None,
+        description='A string for a Honeybee Radiance Modifier (default: None).'
+        )
+
+    modifier_blk: str = Field(
```

Comment:
If you check the current schema of `StateGeometry`, you will see that I decided to call this `modifier_direct` instead of `modifier_blk`. https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/dynamic/stategeo.py#L102-L109